charts/velocity-registry/values.yaml

```diff
@@ -0,0 +1,43 @@
+replicaCount: 1
+
+image:
+  repository: registry
+  tag: 2.8.1
+  pullPolicy: IfNotPresent
+
+env:
+  RegistryAuth:
+    name: REGISTRY_AUTH
+    value: htpasswd
+  RegistryAuthRealm:
+    name: REGISTRY_AUTH_HTPASSWD_REALM
+    value: Registry Realm
+  RegistryAuthPath:
+    name: REGISTRY_AUTH_HTPASSWD_PATH
+    value: /auth/htpasswd
+
+registryContainerPort: 5000
+registryServicePort: 5000
+
+registryResources:
+  limits:
+    cpu: "1"
+    memory: 1Gi
+  requests:
+    cpu: 100m
+    memory: 250Mi
+
+secrets:
+  registryAuthName: registry-auth
+  registryConfigName: registry-docker-config
+  defaultUsername: root
+  defaultPassword: ""  # must be set on installation
```

Comment:
If this is a must you can add a `require` func when used so it will validate it is set, otherwise install will fail